packages/react-link/src/components/Link/useLinkStyles.ts

```diff
@@ -0,0 +1,75 @@
+import { ax, makeStyles } from '@fluentui/react-make-styles';
+import { LinkState } from './Link.types';
+
+const useStyles = makeStyles<LinkState>([
+  // Common styles.
+  [
+    null,
+    tokens => ({
+      backgroundColor: 'transparent',
+      border: 'none',
+      borderBottom: 'solid transparent',
+      borderBottomWidth: tokens.global.strokeWidth.thin,
+      boxSizing: 'border-box',
+      color: tokens.global.palette.brand.primary,
```

Comment:
I think that we should use `.alias` colors instead to ensure that theme switching will work

Reply:
Not too sure here, this IMO should eventually be mapped to component design tokens which can be defined per HC/Dark/light theme

Reply:
I tend to agree with @ling1726 here. Especially since it seems to me like a lot of the things I'm using it for don't match to a specific alias token (i.e. there is no concept of `borderColor` in alias tokens right now).